app/src/main/res/drawable/description_bg_expanded.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <corners
+        android:topLeftRadius="6dp"
+        android:topRightRadius="6dp" />
+    <stroke
+        android:color="?attr/colorBackgroundAccent"
+        android:width="1dp" />
+    <solid android:color="#B3000000" />
+</shape>
```

Comment:
try this for no border at the bottom

```
<?xml version="1.0" encoding="utf-8"?>
<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
    <item>
        <shape>
            <solid android:color="<border color>" />
            <corners
                android:topLeftRadius="6dp"
                android:topRightRadius="6dp" />
        </shape>
    </item>
    <item
        android:left="1dp"
        android:right="1dp"
        android:top="1dp">
        <shape>
            <solid android:color="<body color>" />
            <corners
                android:topLeftRadius="5dp"
                android:topRightRadius="5dp" />
        </shape>
    </item>
</layer-list>
```

Reply:
I will try that but would still be nice to have some indication of the bottom

Reply:
nah it does not work when the body color is transparent

Reply:
Any ideas what to do with it? I'm kinda okay with how it is but maybe it should transform somehow when expanded to full height, idk

Reply:
No idea, I think its fine to leave it like it is.